pkg/storage/loaders/mto_common_solution_system_owner.go

```diff
@@ -0,0 +1,68 @@
+package loaders
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+
+	"github.com/cms-enterprise/mint-app/pkg/appcontext"
+	"github.com/cms-enterprise/mint-app/pkg/models"
+	"github.com/cms-enterprise/mint-app/pkg/storage"
+
+	"github.com/graph-gophers/dataloader/v7"
+)
+
+// mtoCommonSolutionSystemOwnerLoaders holds LoaderWrappers related to MTOCommonSolutionSystemOwner
+type mtoCommonSolutionSystemOwnerLoaders struct {
```

Comment:
these likely need to be updated to allow 1:many and allow nil values (for when somebody hasn't added an owner yet)